### DISCUSSION TITLE
Vote for Multilingual Model Upgrades

### DISCUSSION THREAD

**Discussion by Evezerest** [PaddlePaddle/PaddleOCR#7253] [Polls]
We open this poll to record the multilingual language needs for the following two situations:

1. If you want to add a new model or upgrade the excited language model, please leave a comment describing the usage scenario.

2. If you want to recognize multiple languages by one recognition model, please comment here below, such as `Arabic+Bengali`, 
 
It would be better to provide some images to show the fonts and styles. Note that each language model is able to recognize English. If the result is bad, probably because the size of the training dataset is too small.

We'll update the poll options continuously, if the language model or its combination option you need is already in the poll, please VOTE!

**Comment by matteoPiccoli** [comment]
Korean+English

**Reply by Evezerest** [reply]
Is there any scenario you want to improve? for example, scanned documents or car plate recognition

**Reply by matteoPiccoli** [reply]
Right now I'm working on images, mainly on screenshots coming from android OS. 

According to the language you choose, you could have some sentences with a combination of korean and english words and Paddle OCR doesn't recognize the text properly (this happens for Russian+English and Arabic+English as well). Right now I managed to work around this problem using `language_classification` from PaddleClas, but this solution is quite slow.

**Comment by ahmadSaheb** [comment]
Arabic + English

**Comment by yq-xfl** [comment]
泰语  thai   please

**Comment by clonesangram87** [comment]
Hindi please

**Comment by NSeyedi** [comment]
Persian + Arabic + English

**Comment by Atahanak** [comment]
Turkish please :)

**Comment by tunglambk** [comment]
English + Japanese please

**Comment by asif-ca** [comment]
@Evezerest 
Is it possible to train a speech recognition model on multiple languages such as **English+ Chinese+ Urdu+ Hindi+ Polish+ Bengali+ Japanese+ Arabic and other languages**? 

We are considering fine-tuning the model on all these languages using the ch_PPOCRV3_rec pre-trained model. Can you please advise us on the potential difficulties we might face and how much data we would require for this task?

**Comment by lvyq800** [comment]
Chinese+ tibetan

**Comment by Aminfaraji** [comment]
Persian + Arabic + English

**Comment by gucmurat** [comment]
For Turkish, performance improvement is needed
experiencing difficulty with accurate recognition of Turkish letters (e.g., ç, ı, ğ, ö, ş, ü)

**Reply by turgutguvercin** [reply]
Latin dict has no İ, ğ, Ğ, ş and Ş. Add them to the dictionary and then fine tune it. It improves a lot!
